server/routes/serviceProviderReferrals/showReferralPresenter.ts

```diff
@@ -21,4 +23,49 @@ export default class ShowReferralPresenter {
     { key: 'Name', lines: [`${this.sentBy.firstName} ${this.sentBy.surname}`], isList: false },
     { key: 'Email address', lines: [this.sentBy.email ?? ''], isList: false },
   ]
+
+  readonly serviceUserNotificationBannerArgs = {
+    titleText: 'Service user details',
+    html:
+      `<p class="govuk-notification-banner__heading">${this.serviceUser.firstName} ${this.serviceUser.surname}<p>` +
+      `<p>Date of birth: ${this.serviceUserDateOfBirth}</p>` +
+      `<p class="govuk-body">${this.serviceUserMobile} | ${this.serviceUserEmail}</p>`,
+  }
+
+  private get serviceUserDateOfBirth(): string {
+    const { dateOfBirth } = this.serviceUser
+
+    const notFoundMessage = 'Not found'
+
+    if (dateOfBirth) {
+      const iso8601DateOfBirth = CalendarDay.parseIso8601(dateOfBirth)
+
+      return iso8601DateOfBirth ? ReferralDataPresenterUtils.govukFormattedDate(iso8601DateOfBirth) : notFoundMessage
+    }
+
+    return notFoundMessage
+  }
+
+  private get serviceUserEmail(): string {
+    const { emailAddresses } = this.serviceUser.contactDetails
+
+    if (emailAddresses && emailAddresses.length > 0) {
+      return emailAddresses[0]
+    }
+
+    return 'Email address not found'
```

Comment:
why the difference in 'not found' behaviour for email? the other two use `notFoundMessage`